app/scripts/modules/core/src/artifact/expectedArtifactMultiSelector.component.ts

```diff
@@ -23,7 +23,7 @@ class ExpectedArtifactMultiSelectorComponent implements IComponentOptions {
         <ng-form name="artifacts">
           <div class="col-md-12">
             <div class="form-group">
-              <label class="col-md-3 sm-label-right">{{ctrl.artifactLabel}}</label>
+              <label class="col-md-3 sm-label-right">{{ctrl.artifactLabel}}<help-field key="kubernetes.manifest.requredArtifactsToBind"/></label>
```

Comment:
Can you make these tool tips parameters? This selector is built to be used in other places too (e.g. upcoming rosco & appengine work).

Reply:
Also, typo in the word "required".